tutorials/evoked/30_eeg_erp.py

```diff
@@ -373,7 +374,204 @@
+# ``'eeg'`` channels are used.
+
+# Get peak amplitude and latency.
+tmin, tmax = .065, .115
+ch, lat, amp = l_vis.get_peak(ch_type='eeg', tmin=tmin, tmax=tmax,
+                              mode='pos', return_amplitude=True)
+
+# Convert latency and amplitude to msec and microvolts
+lat *= 1e3
+amp *= 1e6
+
+# Print output
+print(f'Channel: {ch}')
+print(f'Peak Latency: {lat:.3f} msec')
+print(f'Peak Amplitude: {amp:.3f} \u00B5V')
+
+# %%
+# The output shows that the channel ``eeg55`` had the maximum positive peak in
+# the chosen time window. In practice, one might want to pull out the peak for
+# an *a priori* region of interest or a single electrode depending on the study
+# This can be done by combining the `~mne.Evoked.pick`
+# (or `~mne.Evoked.pick_channels`) methods with the `~mne.Evoked.get_peak`
+# method.
+#
+# Here, let's assume we believe the effects of interest will occur
+# at ``eeg59``.
+
+# Get the peak and latency measure from a single ROI
+# Fist, return a copy of l_vis to select the channel from
+l_vis_roi = l_vis.copy().pick('eeg59')
+_, lat_roi, amp_roi = l_vis_roi.get_peak(tmin=tmin, tmax=tmax, mode='pos',
+                                         return_amplitude=True)
+
+# Convert latency and amplitude to msec and microvolts
+lat_roi *= 1e3
+amp_roi *= 1e6
+
+# Print output
+print('Channel: eeg59')
+print(f'Peak Latency: {lat_roi:.3f} msec')
+print(f'Peak Amplitude: {amp_roi:.3f} \u00B5V')
+
+# %%
+# While the peak latency is the same in channels ``eeg55`` and ``eeg59``, the
+# peak amplitudes differ. The above approach can be done on virtual channels
+# created with the `~mne.channels.combine_channels` function and on difference
+# waves created with the `mne.combine_evoked` function (``aud_minus_vis``).
+#
+# While beyond the scope of this tutorial, peak measures are very susceptible
+# to high frequency noise (for discussion, see :footcite:`Luck2014`). One way
+# to avoid this is to apply a non-causal low-pass filters to the ERP. While
+# this can reduce bias in peak amplitude measures due to high frequency
+# noise, it can introduce challenges in interpreting latency measures for
+# effects of interest :footcite:`Rousselet2012,VanRullen2011`.
+#
+# If using peak measures, it is critical to visually inspect the data to
+# make sure the selected time window actually contains a peak. Note that
+# `~mne.Evoked.get_peak` will always identify a peak amplitude in the time
+# window being searched. However, the peak that is identified may be incorrect.
+# Instead of a peak, we could just measure the rising edge of a peak, for
+# instance, which is not ideal. The following demonstrates why visual
+# inspection is crucial. Below, we use a known bad time window (.09 to .12
+# seconds) to search for a peak on ``eeg59``.
+
+# Get BAD peak measures
+bad_tmin, bad_tmax = .09, .12
+_, bad_lat_roi, bad_amp_roi = \
+    l_vis_roi.get_peak(mode='pos', tmin=bad_tmin, tmax=bad_tmax,
+                       return_amplitude=True)
```

Comment:
```suggestion
ch, bad_lat_roi, bad_amp_roi = l_vis_roi.get_peak(
    mode='pos', tmin=bad_tmin, tmax=bad_tmax, return_amplitude=True)
```

we try to avoid backslash-continuation lines if possible.  Also same comment here about storing and printing the channel return value, as a sanity check.